ui/app/routes/vault/cluster/logout.js

```diff
@@ -21,7 +21,7 @@ export default Route.extend(ModelBoundaryRoute, {
     this.namespaceService.reset();
     this.console.set('isOpen', false);
     this.console.clearLog(true);
-    this.clearModelCache();
+    this.replaceWith('vault.cluster.auth', { queryParams: { redirect_to: '' } });
```

Comment:
What's the rationale here? It doesn't seemed associated to the bug to my untrained eyes.

Reply:
I think this is from a bad merge / rebase - it was changed in master recently.

Reply:
yep, that was mistakenly added in during a rebase. good question though!